### ISSUE TITLE
Kotlin exception when attempting to run any lein task from IDEA

### ISSUE THREAD

**Issue by ricardojmendez** [cursive-ide/cursive#1318]
Environment: IDEA 2016.1, Build #IU-145.258, Cursive 1.2.1-2016.1, OS X.

Consistently happening right now, can't start `lein figwheel` nor do `lein run` from inside IDEA.  Cleaning the caches did not help. 

Logging it while I figure out if it's a problem with Cursive or an environment issue.

```
Kotlin reflection implementation is not found at runtime. Make sure you have kotlin-reflect.jar in the classpath
kotlin.jvm.KotlinReflectionNotSupportedError: Kotlin reflection implementation is not found at runtime. Make sure you have kotlin-reflect.jar in the classpath
	at kotlin.jvm.internal.ClassReference.error(ClassReference.kt:45)
	at kotlin.jvm.internal.ClassReference.getSimpleName(ClassReference.kt:25)
	at cursive.leiningen.run.LeiningenCommandLineState.<init>(LeiningenCommandLineState.kt:39)
	at cursive.leiningen.run.LeiningenRunConfiguration.getState(LeiningenRunConfiguration.kt:36)
	at com.intellij.execution.runners.ExecutionEnvironment.getState(ExecutionEnvironment.java:158)
	at com.intellij.execution.runners.BaseProgramRunner.execute(BaseProgramRunner.java:55)
	at com.intellij.execution.runners.GenericProgramRunner.execute(GenericProgramRunner.java:31)
	at com.intellij.execution.runners.BaseProgramRunner.execute(BaseProgramRunner.java:50)
	at com.intellij.execution.runners.GenericProgramRunner.execute(GenericProgramRunner.java:31)
	at com.intellij.execution.ProgramRunnerUtil.executeConfiguration(ProgramRunnerUtil.java:121)
	at com.intellij.execution.impl.ExecutionManagerImpl.a(ExecutionManagerImpl.java:116)
	at com.intellij.execution.impl.ExecutionManagerImpl.access$400(ExecutionManagerImpl.java:67)
	at com.intellij.execution.impl.ExecutionManagerImpl$7.run(ExecutionManagerImpl.java:516)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:378)
	at com.intellij.util.Alarm$Request.run(Alarm.java:398)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:227)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.a(IdeEventQueue.java:857)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:658)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:386)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by ricardojmendez** [comment]
Notes so far:
- This is happening on both IDEA 15 and 2016.1
- I found kotlin-reflect.jar on both applications
- I've tried re-installing IDEA, in case something had gotten corrupted
- Cleaned caches for both to no avail
- The plugin for Kotlin support said it could be updated, which I've tried. Currently on 1.0.1-release-IJ143-27. 

The issue persists. REPLs start just fine, it seems to be specifically related to lein.
